R/create_default_parameters.R

```diff
@@ -251,6 +251,10 @@ create_default_Population <- function(
   n_years <- get_n_years(data)
   n_ages <- get_n_ages(data)
 
+  # Define age and year vectors needed for the rep() calls ===
+  ages <- get_ages(data)
+  years <- get_start_year(data):get_end_year(data)
```

Comment:
Is this other work that snuck in? It looks like there are a few commits that are unrelated to dependabot and updating the github action version in files. I didn't review these, but can if it is intended to be in this PR.